### PR TITLE
Bump develop version to 3.5.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by rohitsinha54**

### CODE REVIEW

**Comment by bdmogal** [comment]
Are you sure this updates it everywhere? Remember seeing over 90 files during the last release.

**Comment by rohitsinha54** [comment]
Here is the last release PR which I referenced there are 83 files 
https://github.com/caskdata/cdap/pull/4711/files

**Comment by bdmogal** [comment]
Refer to this instead - https://github.com/caskdata/cdap/pull/5187

File: cdap-distributions/src/packer/cdap-sdk-ubuntu12-with-uri.json [hunk 1]
```diff
@@ -126,7 +126,7 @@
       "skip_install": true,
       "json": {
         "cdap": {
-          "version": "3.4.0-SNAPSHOT-1",
+          "version": "3.5.0-SNAPSHOT-1",
```
Comment (rohitsinha54) [inline cdap-distributions/src/packer/cdap-sdk-ubuntu12-with-uri.json]: Why is this -1 ?

Comment (anwar6953) [inline cdap-distributions/src/packer/cdap-sdk-ubuntu12-with-uri.json]: @dereklwood @wolf31o2 ?

Comment (bdmogal) [inline cdap-distributions/src/packer/cdap-sdk-ubuntu12-with-uri.json]: It is probably the ``release`` field. But I'll let @dereklwood @wolf31o2 confirm

**Comment by bdmogal** [comment]
Apart from your question, LGTM. Please run a local maven build just to be sure.

**Comment by rohitsinha54** [comment]
Build passes locally

**Pull request merged**